### DISCUSSION TITLE
Time Info Missing In Landscape Mode

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#78] [Feature Requests]
@rern,

My old 7" tablet does not display the playback mode and time info in landscape, as it used to in previous versions. The display is 1024x600.

If this is a permanent feature change and not a bug, I would rather see the time figures than the progress bar. Would it be possible to restore the mode/time line when the progress bar is deselected?

I know the mode/time info is restored if the top/bottom bars are hidden, but tapping is much easier than swiping when the tablet is on a stand.

**Comment by rern** [comment]
Post your screenshot with your device viewport width and height.
(Get it from: https://www.mydevice.io/)

**Reply by R101G** [reply]
mydevice.io shows 1024x600

Sometimes this:
![Screenshot_2021-01-05-18-13-56](https://user-images.githubusercontent.com/75951837/103684515-9b50b580-4f83-11eb-84b9-4a3a46d72f79.png)

Sometimes this:
![Screenshot_2021-01-05-18-18-13](https://user-images.githubusercontent.com/75951837/103684564-ac99c200-4f83-11eb-9b71-30db3ca07fd6.png)

Or, with no top/bottom bars:
![Screenshot_2021-01-05-18-18-53](https://user-images.githubusercontent.com/75951837/103684627-c89d6380-4f83-11eb-9973-6ab648d31f9a.png)

I think the different displays depend on what page you return from, but I have not worked out the pattern.

**Reply by R101G** [reply]
@rern,

Thanks rern. I now just lose the file info in landscape, which is a good compromise, and I have not seen any inconsistencies. Would it be possible to move the track number up to the mode/time line, though?
